hircluster.c

```diff
@@ -86,6 +86,11 @@ typedef struct cluster_async_data {
     void *privdata;
 } cluster_async_data;
 
+typedef struct event_callback {
+    void (*fn)(const struct redisClusterContext *cc, int event, void *privdata);
+    void *privdata;
+} event_callback_t;
```

Comment:
"Names that end with ‘_t’ are reserved for additional type names" by the POSIX standard, so we shouldn't use the suffix. See https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html

Reply:
Ah thanks, forgot about that